extension/src/content/index.js

```diff
@@ -186,228 +203,28 @@ function init() {
-          const emptyLineRegex = /^\s*$/u;
-          const indentedCodeBlockRegex = /^\s{4}/u;
-          const codeSnippetRegex = /`{1,2}[^\n`]+`{1,2}/u;
-          const lastGroup = groups[groups.length - 1];
-
-          /*
-           * Code blocks using backicks or tildes:
-           *
-           * ```lang-js
-           * const foo = 'bar';
-           * ```
-           *
-           * ~~~lang-js
-           * const foo = 'bar';
-           * ~~~
-           */
-          if (codeBlockRegex.test(line)) {
-            if (isInBlock) {
-              lastGroup.push(line);
-              isInBlock = false;
-            } else {
-              groups.push([line]);
-              isInBlock = true;
-            }
-          } else if (isInBlock) {
-            lastGroup.push(line);
-
-            /*
-             * Code snippet using backicks:
-             *
-             * `const foo = 'bar';`
-             *
-             * ``const foo = `${bar}`;``
-             */
-          } else if (codeSnippetRegex.test(line)) {
-            groups.push([line]);
-
-            /*
-             * Code blocks using indented lines:
-             *
-             *     const foo = 'bar';
-             *     console.log(typeof foo);
-             *
-             * <!-- language: lang-js -->
-             *
-             *     const foo = 'bar';
-             *     console.log(typeof foo);
-             */
-          } else if (emptyLineRegex.test(line)) {
-            if (
-              lastGroup &&
-              indentedCodeLangRegex.test(lastGroup[lastGroup.length - 1])
-            ) {
-              lastGroup.push(line);
-            }
-          } else if (indentedCodeLangRegex.test(line)) {
-            groups.push([line]);
-          } else if (indentedCodeBlockRegex.test(line)) {
-            if (
-              lastGroup &&
-              (indentedCodeBlockRegex.test(lastGroup[lastGroup.length - 1]) ||
-                emptyLineRegex.test(lastGroup[lastGroup.length - 1]))
-            ) {
-              lastGroup.push(line);
-            } else {
-              groups.push([line]);
-            }
-            /*
-             * language-all comments:
-             *
-             * <!-- language-all: lang-js -->
-             */
-          } else if (langAllRegex.test(line)) {
-            groups.push([line]);
-          }
-
-          return groups;
-        }, []);
-
-        // There is an unclosed code block.
-        if (isInBlock) {
-          return;
-        }
-
-        if (codeBlocks.length) {
-          let langAll = null;
-
-          // https://stackoverflow.com/editing-help#syntax-highlighting
-          codeBlocks.forEach(lines => {
-            const codeBlockRegex = /^\s{0,3}(?:```|~~~)\s*(?:lang-(.+))?/u;
-            const indentedCodeWithLangRegex = /^\s*<!-- language: lang-(.+) -->/u;
-            const langAllRegex = /^\s*<!-- language-all: lang-(.+) -->/u;
```

Comment:
Looks like we're missing the `language-all` and code snippet formatting behavior that was added [here](https://github.com/prettier/prettier-chrome-extension/commit/0407f000ce6632eb2fc54adb7b2672d8efafc0b6). Maybe from a bad merge conflict resolution? Might be worth copying over the entire click event handler from `master` starting from https://github.com/prettier/prettier-chrome-extension/pull/49/files#diff-2b5769826b6670c8ab28f63985a301a8L210.